src/ipc/StoreMap.cc

```diff
@@ -268,6 +268,8 @@ Ipc::StoreMap::freeEntry(const sfileno fileno)
         return result;
     }
 
+    // TODO: check whether 'empty' anchors can be still locked.
```

Comment:
Yes, any anchor/entry can be locked. Locking does not depend on the entry state.